src/azul/terraform.py

```diff
@@ -816,22 +816,42 @@ def tf_config(self, app_name):
-            f'DEFAULT_{response_type}': {
-                'responseParameters': {
-                    # Static value response header parameters must be enclosed
-                    # within a pair of single quotes.
-                    #
-                    # https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html#mapping-response-parameters
-                    # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-gateway-responses.html
-                    #
-                    # Note that azul.strings.single_quote() is not used here
-                    # since API Gateway allows internal single quotes in the
-                    # value, which that function would prohibit.
-                    #
-                    f'gatewayresponse.header.{k}': f"'{v}'"
-                    for k, v in AzulChaliceApp.security_headers.items()
-                }
-            } for response_type in ['4XX', '5XX']
+            **{
+                f'DEFAULT_{response_type}': {
+                    'responseParameters': {
+                        # Static value response header parameters must be enclosed
+                        # within a pair of single quotes.
+                        #
+                        # https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html#mapping-response-parameters
+                        # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-gateway-responses.html
+                        #
+                        # Note that azul.strings.single_quote() is not used here
+                        # since API Gateway allows internal single quotes in the
+                        # value, which that function would prohibit.
+                        #
+                        f'gatewayresponse.header.{k}': f"'{v}'"
+                        for k, v in AzulChaliceApp.security_headers.items()
+                    }
+                } for response_type in ['4XX', '5XX']
+            },
+            **{
+                response_type: {
+                    'responseParameters': {
+                        **{
+                            f'gatewayresponse.header.{k}': f"'{v}'"
+                            for k, v in AzulChaliceApp.security_headers.items()
+                        },
+                        'gatewayresponse.header.Retry-After': "'10'"
+                    },
+                    'responseTemplates': {
+                        "application/json": json.dumps({
+                            'message': '504 Gateway Timeout. Wait the number of'
+                                       ' seconds given in the `Retry-After`'
+                                       ' header before retrying the request.'
+                        })
+                    }
+                } for response_type in ['INTEGRATION_TIMEOUT', 'INTEGRATION_FAILURE']
+            }
```

Comment:
Consider employing the changes in the following patch, make the diff smaller.
```diff
Index: src/azul/terraform.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/azul/terraform.py b/src/azul/terraform.py
--- a/src/azul/terraform.py	(revision 1f2033d1536b4b60bc2d1632e597fb3c74aa2675)
+++ b/src/azul/terraform.py	(date 1728947754159)
@@ -816,7 +816,7 @@
         openapi_spec[key] = config.minimum_compression_size
         assert 'aws_api_gateway_gateway_response' not in resources, resources
         openapi_spec['x-amazon-apigateway-gateway-responses'] = {
-            **{
+            {
                 f'DEFAULT_{response_type}': {
                     'responseParameters': {
                         # Static value response header parameters must be enclosed
@@ -833,8 +833,7 @@
                         for k, v in AzulChaliceApp.security_headers.items()
                     }
                 } for response_type in ['4XX', '5XX']
-            },
-            **{
+            } | {
                 response_type: {
                     'responseParameters': {
                         **{

```

Reply:
PyCharm wants to format
```
some_variable = {
    'foo': 'FOO'
} | {
   'bar': 'BAR'
}
```

as
```
some_variable = {
                    'foo': 'FOO'
                } | {
                    'bar': 'BAR'
                }
```

so I wrapped it in parens to get
```
some_variable = (
    {
        'foo': 'FOO'
    } | {
        'bar': 'BAR'
    }
)
```